Topics/Tech_Stacks/React.md

```diff
@@ -1,37 +1,59 @@
-https://learn.microsoft.com/en-us/windows/dev-environment/javascript/
+[Front-End Mentor](https://www.frontendmentor.io/)
+These frontend challenges for HTML, CSS and JavaScript will help you review core skills while also making interesting projects.
 
+[Mozilla React Tutorial](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started)
+React tutorial by MDN Web Docs. Useful readings for if you have time, but not necessary by any means.
 
-1 hour in depth tutorials about React. These Youtube crash course provides a great in depth look at React and the various important components that you will be using most often. Very useful if you are learning React under a strict time limit for whatever reason. \
-https://www.youtube.com/watch?v=w7ejDZ8SWv8 \
-https://www.youtube.com/watch?v=b9eMGE7QtTk 
+[Microsoft JavaScript Tutorial](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/)
+Javascript frameworks tutorial provided by Microsoft. Useful readings, but not necessary to learn React. 
 
+### Useful React Tools and Libraries
 
-Online development environment to try React out in (Among other frontend tools). Great for testing the waters with React to see if it will be something you want to learn or not. \
-https://codepen.io/
+[Create React App](https://create-react-app.dev/)
+Allows you to set up your React applications using a single command without having to configure many build tools. Provides a lot of useful functionality like instant reloads and optimizations for deployment. 
 
+[usehooks-ts](https://usehooks-ts.com/)
+A React library of useful hooks that go well beyond those built into React. Don't reinvent the wheel! usehooks offers over forty ready-to-use hooks such as [useEffectOnce](https://usehooks-ts.com/react-hook/use-effect-once) and [useInterval](https://usehooks-ts.com/react-hook/use-interval). 
 
-Host your React websites straight from your Github repo. Website hosting is fairly expensive, so this is here as a free alternative for website deployment. \
-https://pages.github.com/
+[Redux](https://redux.js.org/)
+The official React solution for state management. Redux provides functionality for reducers, advanced debugging tools, and more that will simplify the state management for your app.
 
+[Material UI](https://mui.com/material-ui/)
+A fully-featured UI library that offers a huge range of common components that are easily styled and based on Google's Material Design.
+
+[Bootstrap](https://getbootstrap.com/)
+Another responsive UI library that allows you to focus on development by handling the design logistics for you.  
+
+### Miscellaneous
+
+[GitHub Pages](https://pages.github.com/)
+Host your React websites straight from your Github repo. Website hosting is fairly expensive, so this is here as a free alternative for website deployment for simple applications.
```

Comment:
I'm nitpicking but I don't think website hosting is too expensive, so maybe we can reword this to just say "If you don't want to spend money on website hosting" or "If you don't need server functionality and want to host for free" or something like that.

Reply:
Fair point

Reply:
It might be a good idea to plug the page I contributed (GitHub pages). I am a bit unsure on how to do that as it is not committed yet.

Reply:
I'll add a link to the index page, once you commit I can add the link if there's time.